scripts/dockerNodeStats/main.py

```diff
@@ -0,0 +1,197 @@
+import csv
+import json
+import logging
+import os
+import subprocess
+from collections import defaultdict
+
+import tqdm
+
+# Get log level from environment variable
+log_level: str = os.environ.get("LOG_LEVEL", "INFO").upper()
+logging.basicConfig(level=log_level)
+
+# Create logger
+logger = logging.getLogger(__name__)
+
+# Create handler
+handler = logging.StreamHandler()
+
+# Create formatter and add to handler
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+handler.setFormatter(formatter)
+
+# Add handler to logger
+logger.addHandler(handler)
+logger.propagate = False
+
+# Use logger
+logger.info("Starting...")
+service_ids: list[str] = (
```

Comment:
I would avoid running everything on global level since this is a bad practice. A good practice is to have `if __name__ == "__main__": ...` and further function calls.

However, in this particular case it is clearly not a must.